package.json

```diff
@@ -183,7 +184,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "^1.18.0",
+    "@department-of-veterans-affairs/formation": "^2.0.0-fa",
```

Comment:
Just noting that this is a temporary version and will point to `2.0.0` once we're ready.

Reply:
Looks like we still need to uninstall the old Font Awesome, https://github.com/department-of-veterans-affairs/vets-website/pull/9369/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L203.

Can we also move `@fortawesome/fontawesome-free` down to `dependencies`? Right now it's listed under `devDependencies`. The distinction is sort of unclear, but we try to follow this general rule - 

> The difference between these two, is that devDependencies are modules which are only required during development, while dependencies are modules which are also required at runtime.

[article](https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c)

Reply:
I removed Font Awesome 4 from `dependencies` and moved Font Awesome 5 from `devDependencies` to `dependencies`